packages/SwingSet/src/devices/vat-admin/device-vat-admin.js

```diff
@@ -156,6 +156,7 @@ export function buildDevice(tools, endowments) {
 
         // D(devices.vatAdmin).createMeter(remaining, threshold) -> meterID
         if (method === 'createMeter') {
+          /** @type {any} */
```

Comment:
Does this seem correct? If not, should it have XXX?

array destructuring 2 lines down strongly suggests `Array<something>`.
`unknown[]` would be best - i.e. "tsc, please gripe if I have made any assumptions about the contents of this array."

Reply:
I time box types improvements in swingset packages because they're usually a rabbit hole

Reply:
This was a vestige of some earlier version of `unserialize`. since `argsCapData` is any `args` is too